### PR TITLE
Adding lact for archlinux

### PR DESCRIPTION
**Pull request by Nightmare1984**
# Pull Request

## Title
Adding Lact For ArchLinux And Adding Aur Helper Checker 

## Type of Change
- [x] New feature
- [ ] Bug fix
- [ ] Documentation Update
- [ ] Refactoring
- [ ] Hotfix
- [ ] Security patch
- [ ] UI/UX improvement

## Description
This PR Add's Amd Gpu Overclocking app And Aur Checker To Help Install Aur Package's In Linutil 

## Testing
Just Press Lact Setup For ArchLinux

## Impact
Overclocking And A Aur Checker For App's

## Additional Information
Adding Aur Checker To Make Lact Install Work

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Review by ChrisTitusTech — CHANGES_REQUESTED** [review]
This is an agnostic common script run by debian and rhel systems. your checkenv changes would break them.

**Comment by nnyyxxxx** [comment]
@Nightmare1984 remove lines 20-14 as its not need with the merge of #217

**Review by nnyyxxxx — CHANGES_REQUESTED** [review]
@Nightmare1984  remove lines 20-14 as its not needed with the merge of #217

File: src/commands/applications-setup/lact-setup.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    echo "Installing Lact"
+    if command_exists lact; then
+        echo "Lact is already installed."
+        return
+    fi
+    checkAURHelper
+    $AUR_HELPER -S --noconfirm lact
+    if command_exists systemctl; then
+       $ESCALATION_TOOL systemctl enable --now lactd
+    fi
```
Comment (nnyyxxxx) [inline src/commands/applications-setup/lact-setup.sh:15]: No need for this if statement here, delete line 13, 15, and leave line 14 untouched.

Comment (nnyyxxxx) [reply src/commands/applications-setup/lact-setup.sh:15]: @Nightmare1984 Please add support for more distros, take advice from the other scripts.

File: src/commands/applications-setup/lact-setup.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+. ../common-script.sh
+
+setuplact() {
+    echo "Installing Lact"
+    if command_exists lact; then
+        echo "Lact is already installed."
+        return
+    fi
+    checkAURHelper
```
Comment (nnyyxxxx) [inline src/commands/applications-setup/lact-setup.sh:11]: "checkAURHelper" needs to be moved under "checkEnv"

File: src/commands/applications-setup/lact-setup.sh [hunk 3]
```diff
@@ -0,0 +1,49 @@
+#!/bin/sh -e
+
+. ../common-script.sh
+
+setuplact() {
+    echo "Installing Lact"
+    if command_exists lact; then
+        echo "Lact already Installed"
```
Comment (nnyyxxxx) [inline src/commands/applications-setup/lact-setup.sh:8]:
```suggestion
        echo "Lact already Installed"
        exit 1
```

**Review by nnyyxxxx — APPROVED** [review]
Looks good, just make sure you have tested this.

**Comment by Nightmare1984** [comment]
Lact rpm fedora file is broken from maker's GitHub release i Will Try to see if i can make it compile from source For Fedora user's To Fix The Issue.

**Comment by Nightmare1984** [comment]
i still got some work to do for other distro's github api not fetching the file right in the code

**Comment by nnyyxxxx** [comment]
@Nightmare1984 Mark this PR as a draft until you figure it out.

**Comment by ChrisTitusTech** [comment]
Sorry for the inconvenience. We had a massive restructure of the codebase to improve future development. Because of this can you update your PR to the new structure. Thank you for your assistance and contribution.

**Pull request closed without merging**